locale/en/messages.json

```diff
@@ -100,6 +100,7 @@
   "Application Availability and Reachability per Minute": "",
   "Application Reachability": "",
   "Apply": "",
+  "Are you sure you want to delete this LDAP directory?": "",
```

Comment:
This is part of @Poltergeist 's https://github.com/dcos/dcos-ui/pull/3835 and should disappear once his PR is merged.